hadoop-ozone/httpfsgateway/pom.xml

```diff
@@ -111,26 +111,37 @@
       <artifactId>slf4j-reload4j</artifactId>
       <scope>runtime</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.curator</groupId>
-      <artifactId>apache-curator</artifactId>
-      <version>2.4.0</version>
-      <type>pom</type>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.curator</groupId>
-      <artifactId>curator-test</artifactId>
-      <version>2.4.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.curator</groupId>
-      <artifactId>curator-client</artifactId>
-      <version>2.4.0</version>
-    </dependency>
     <dependency>
       <groupId>org.apache.curator</groupId>
       <artifactId>curator-framework</artifactId>
+      <scope>runtime</scope>
       <version>2.4.0</version>
+      <exclusions>
```

Comment:
Can you add a comment as to why these dependencies were excluded.

Reply:
thanks for the review @kerneltime, I updated my patch with a comment, that I excluded these as non of them is used by the HttpFS module, but all of them would be a new (unnecessary) dependency in Ozone, the `jar-report.txt` should be updated with that too. I didn't add a comment for each, but I will detail it here.

- the `commons-math` (we are using `commons-math3` in Ozone) and `jline` is never used in the HttpFS module, even in `apache/hadoop`'s HttpFS module (`jline` is used in `hadoop-auth` there, but that was removed from here in #2536). 

- `javax.mail` and `org.jboss.netty` is excluded in `apache/hadoop` as well.

- `log4j` and `slf4j-log4j12` is not used in Ozone anymore, @adoroszlai replaced them in #3773, I use `reload4j` like he did. 

I checked both projects dependency trees to verify these informations. let me know if something is not clear :)